corehq/apps/hqwebapp/forms.py

```diff
@@ -55,18 +55,16 @@ def clean(self):
         if user and user.is_locked_out():
             metrics_counter('commcare.auth.lockouts')
             raise ValidationError(LOCKOUT_MESSAGE)
-        try:
-            cleaned_data = super(EmailAuthenticationForm, self).clean()
-            if not user:
-                try:
-                    _ = ConsumerUser.objects.get(user=self.user_cache)
-                    EmailAuthenticationForm.create_web_user_from_consumer_user(username,
-                                                                               password,
-                                                                               get_ip(self.request))
-                except ConsumerUser.DoesNotExist:
-                    self.get_invalid_login_error()
-        except ValidationError:
-            raise
+
+        cleaned_data = super(EmailAuthenticationForm, self).clean()
+        if not user:
+            consumer_user = ConsumerUser.objects.get_or_none(user=self.user_cache)
```

Comment:
I think this should still use the normal `get` and `try/except`

Reply:
Made the change